packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputCurrency.tsx

```diff
@@ -31,22 +40,31 @@ const CoinmarketFormInputCurrency = ({
 
     const fiatCurrencies = getFiatCurrenciesProps(context);
     const currencies = fiatCurrencies?.supportedFiatCurrencies ?? null;
-    const options = currencies
-        ? [...currencies].map(currency => buildFiatOption(currency))
-        : buildCurrencyOptions(defaultCurrency);
+    const options = useMemo(
+        () =>
+            currencies
+                ? [...currencies].map(currency => buildFiatOption(currency))
```

Comment:
Just use `currencies`

Reply:
It is `Set`, which is why there is a destructor.